subsys/bluetooth/controller/Kconfig

```diff
@@ -6,6 +6,24 @@
 
 menu "Nordic BLE controller"
 
+# BT_HCI_TX_STACK_SIZE is declared in Zephyr and also here for a second time,
+# for set the valid default size of the HCI Tx stack.
+config BT_HCI_TX_STACK_SIZE
+	# NOTE: This value is derived from other symbols and should only be
+	# changed if required by architecture
+	int "HCI Tx thread stack size"
```

Comment:
See https://github.com/zephyrproject-rtos/zephyr/blob/master/subsys/bluetooth/host/Kconfig#L66
If the user first builds a sample with another LL, then selects BT_LL_NRFXLIB the default will not change.
@joerchan, do you know when this will get downstreamed? We should select this option for the BT_LL_NRFXLIB, users shouldn't really need to mess with this config..

Reply:
This works only when you put CONFIG_BT_LL_NRFXLIB=y in prf.conf file I do not find any solution to change it automatically when you change controller from menuconfig.

Reply:
The next upmerge is scheduled to after zephyr 2.1 and the multi-image rework which is in progress. So I'm not sure when that would happen but not in the immediate future.
@thomasstenersen If this commit is needed we could cherry-pick it instead. Should we do that?

Reply:
You could try to remove the prompt or we'll cherry-pick the commit and updating the Kconfig for the BT_LL_NRFXLIB  to select no prompt. I don't have a strong preference. The commit from upstream will be downstream eventually, so might be easiest to cherry-pick?